### ISSUE TITLE
Is Zope only supported in Python 2?

### ISSUE THREAD

**Issue by jgeorgeson** [cobrateam/splinter#589]
I'm seeing an exception using `Browser('zope.testbrowser')`

```
Traceback (most recent call last):
  File "/d01/sandboxes/jgeorgeson/git/Dev-Engineering/gitlab-scripts/python-gitlab/lib64/python3.6/site-packages/splinter/browser.py", line 60, in Browser
    driver = _DRIVERS[driver_name]
KeyError: 'zope.testbrowser'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./scripts/import-project.py", line 5, in <module>
    with Browser('zope.testbrowser') as browser:
  File "/d01/sandboxes/jgeorgeson/git/Dev-Engineering/gitlab-scripts/python-gitlab/lib64/python3.6/site-packages/splinter/browser.py", line 62, in Browser
    raise DriverNotFoundError("No driver for %s" % driver_name)
splinter.exceptions.DriverNotFoundError: No driver for zope.testbrowser
```

And see the sys.version_info check [here](https://github.com/cobrateam/splinter/blob/2eec5aa844f743ad3dd4ffc4a17070f5a5b7d9b5/splinter/browser.py#L25)

```
$ python --version
Python 3.6.3
$ pip3 list splinter[zopetestbrowser]
DEPRECATION: The default format will switch to columns in the future. You can use--format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
beautifulsoup4 (4.6.0)
certifi (2018.1.18)
chardet (3.0.4)
cssselect (1.0.3)
idna (2.6)
lxml (4.1.1)
pip (9.0.1)
python-gitlab (1.2.0)
pytz (2018.3)
PyYAML (3.12)
requests (2.18.4)
selenium (3.9.0)
setuptools (28.8.0)
six (1.11.0)
splinter (0.7.7)
urllib3 (1.22)
waitress (1.1.0)
WebOb (1.7.4)
WebTest (2.0.29)
WSGIProxy2 (0.4.4)
zope.cachedescriptors (4.3.1)
zope.event (4.3.0)
zope.interface (4.4.3)
zope.schema (4.5.0)
zope.testbrowser (5.2.4)
```

**Comment by andrewsmedina** [comment]
Currently zope is only supported in Python 2

**Comment by boyska** [comment]
Is this a zope bug, or a splinter bug? according to https://pypi.org/project/zope.testbrowser/
zope.testbrowser is supported on py 3.5+

**Comment by andrewsmedina** [comment]
Currently zope driver with splinter works only for python 2

**Comment by jsfehler** [comment]
Fixed by #771
